sites/gcweb-menu/_base.scss

```diff
@@ -29,18 +29,22 @@
  *
  */
 
-.gcweb-menu {
+#wb-bnr + .gcweb-menu {
```

Comment:
This is for the deprecated pattern, please add a comment saying that before the selector